benchmarks/setup/cloud-default/cloudcredentials.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-credentials
+type: Opaque
+stringData:
+  clientId: hH55UFivfw-bbHAuPwN545oyv8tTdW0z
```

Comment:
Are these actual credentials?

Reply:
Example ones; I already deleted the client :wink: Wanted to make the example as real as possible otherwise it is often confusing where to put and how the values.

Reply:
Can we add a comment on the README to make sure nobody actually commits real credentials :sweat_smile: ?

Reply:
Of course we can :D I think nobody does that, but even when it is just a small scope right :D

Reply:
I'm thinking, maybe someone sees the example and thinks "oh it's safe to commit them I guess"? :man_shrugging:

Reply:
ok done

Reply:
ah ok missed your comment :D should I add this comment to the default secret as well ?

Reply:
Hm, or can we .gitignore it? I think if the original is already committed then it won't be removed, but no further ones would be committed?

Reply:
Ok lets try that

Reply:
ok works